include/nbl/builtin/hlsl/property_pool/transfer.hlsl

```diff
@@ -49,6 +52,9 @@ struct GlobalPushContants
 
 NBL_CONSTEXPR uint32_t MaxPropertiesPerDispatch = 128;
 
+// TODO: instead use some sort of replace function for getting optimal size?
+NBL_CONSTEXPR uint32_t OptimalDispatchSize = 256;
```

Comment:
you can use the device JIT to query the max compute dispatch size, I'd round it down to nearest PoT though, so the divisions aren't expensive